bleach/__init__.py

```diff
@@ -18,9 +18,9 @@
 
 
 # yyyymmdd
-__releasedate__ = '20200429'
+__releasedate__ = '20200515'
```

Comment:
I only set the date here for releases. When it's in development, I leave this as the empty string.